src/style.css

```diff
@@ -0,0 +1,136 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: flex;
+  height: 134px;
+  width: 160px;
+  margin: 32px 19px auto;
+}
+
+.card__model {
+  display: flex;
+  color: #060b35;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+
+  margin: 40px 16px 0;
+}
+
+.card__code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-top: 4px;
+  margin-left: 16px;
+  margin-bottom: 0;
+}
+
+.review {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-content: center;
+  align-items: center;
+  margin: 16px;
+  margin-bottom: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.star {
+  display: block;
+  margin: 0;
+  padding: 0;
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--1 .star:nth-child(-n + 4) {
```

Comment:
You should have all your styles from "stars" task, so you can change amount of stars on your card only with changing a modifier on your stars block from html page. Checklist P.4.